XLS-0065d-single-asset-vault/README.md

```diff
@@ -0,0 +1,651 @@
+- Delete the `AccountRoot` object of the _pseudo-account_, and its `DirectoryNode` objects.
+- Delete the `Vault` object.
+
+##### 3.1.3.3 Invariants
+
+**TBD**
+
+[**Return to Index**](#index)
+
+### 3.2 `VaultDeposit` and `VaultWithdraw` transactions
+
+Depositors call the `VaultDeposit` and `VaultWithdraw` transactions to add or remove assets from the Tokenized Vault.
+
+[**Return to Index**](#index)
+
+#### 3.2.1 `VaultDeposit` transaction
+
+The `VaultDeposit` transaction adds Liqudity in exchange for vault shares.
+
+| Field Name        |     Required?      |      JSON Type       | Internal Type | Default Value | Description                                            |
+| ----------------- | :----------------: | :------------------: | :-----------: | :-----------: | :----------------------------------------------------- |
+| `TransactionType` | :heavy_check_mark: |       `string`       |   `UINT16`    |     `61`      | Transaction type.                                      |
+| `VaultID`         | :heavy_check_mark: |       `string`       |   `HASH256`   |     `N/A`     | The ID of the vault to which the assets are deposited. |
+| `Amount`          | :heavy_check_mark: | `string` or `object` |   `AMOUNT`    |     `N/A`     | Asset amount to deposit.                               |
+
+##### 3.2.1.1 Failure conditions
+
+- `Vault` object with the `VaultID` does not exist on the ledger.
+- The asset type of the vault does not match the asset type the depositor is depositing.
+- The depositor does not have sufficient funds to make a deposit.
+- Adding the `Amount` to the `AssetsTotal` of the vault would exceed the `AssetsMaximum`.
+- The `Vault` `lsfPrivate` flag is set and the `Account` depositing the assets does not have credentials in the permissioned domain.
+
+- The `Vault.Asset` is `MPT`:
+
+  - `MPTokenIssuance.lsfMPTCanTransfer` is not set (the asset is not transferable).
+  - `MPTokenIssuance.lsfMPTLocked` flag is set (the asset is globally locked).
+  - `MPToken(MPTokenIssuanceID, AccountID).lsfMPTLocked` flag is set (the asset is locked for the depositor).
+  - `MPToken(MPTokenIssuanceID, AccountID).MPTAmount` < `Amount` (insufficient balance).
+
+- The `Asset` is an `IOU`:
+  - The `lsfGlobalFreeze` flag is set on the issuing account (the asset is frozen).
+  - The `lsfHighFreeze` or `lsfLowFreeze` flag is set on the `RippleState` object between the Asset `Issuer` and the depositor.
+  - The `RippleState` object `Balance` < `Amount` (insufficient balance).
+
+##### 3.2.1.2 State Changes
+
+If no `MPToken` object exists for the depositor, create one. For object details, see [3.4.2 `MPToken`](#3.4.2-MPToken).
+
+- Increase the `MPTAmount` field of the share `MPToken` object of the `Account` by $\Delta_{share}$.
+- Increase the `OutstandingAmount` field of the share `MPTokenIssuance` object by $\Delta_{share}$.
+- Increase the `SharesTotal` of the `Vault` object by $\Delta_{share}$.
+
+- Increase the `SharesTotal` by $\Delta_{share}$.
```

Comment:
Are these the same?

Reply:
Yep, they are. Good spot. Will remove the duplicate.